src/include/common/enums/rel_direction.h

```diff
@@ -1,21 +1,11 @@
 #pragma once
 
 #include <cstdint>
-#include <string>
-#include <vector>
 
 namespace kuzu {
 namespace common {
 
-enum class RelDataDirection : uint8_t { FWD = 0, BWD = 1 };
-
-struct RelDataDirectionUtils {
-    static inline std::vector<RelDataDirection> getRelDataDirections() {
-        return std::vector<RelDataDirection>{RelDataDirection::FWD, RelDataDirection::BWD};
-    }
-
-    static std::string relDataDirectionToString(RelDataDirection direction);
-};
+enum class RelDataDirection : uint8_t { FWD = 0, BWD = 1, INVALID = 2 };
```

Comment:
When will the relDirection be INVALID? i think it should be either FWD or BWD, Invalid is a little bit confusing

Reply:
It was supposed to be a safe guard for assertions. but I don't think it's necessary, removed it now.